Tools/AP_Periph/can.cpp

```diff
@@ -583,6 +583,32 @@ static void set_rgb_led(uint8_t red, uint8_t green, uint8_t blue)
         dev->transfer(&v, 1, nullptr, 0);
     }
 #endif // HAL_PERIPH_ENABLE_NCP5623_BGR_LED
+#ifdef HAL_PERIPH_ENABLE_TOSHIBA_LED
+{
+#define TOSHIBA_LED_PWM0    0x01    // pwm0 register
+#define TOSHIBA_LED_ENABLE  0x04    // enable register
+#define TOSHIBA_LED_I2C_ADDR 0x55    // default I2C bus address
+
+        static AP_HAL::OwnPtr<AP_HAL::I2CDevice> dev;
+        if (!dev) {
+            dev = std::move(hal.i2c_mgr->get_device(0, TOSHIBA_LED_I2C_ADDR));
+        }
+        WITH_SEMAPHORE(dev->get_semaphore());
+        dev->set_retries(10);
```

Comment:
lets reduce this to 1

``dev->set_retries(1);``

In a different PR I'd like to change the other two drivers to (1) as well

Reply:
Maybe not. The `AP_Notify` code had it at 10, but in an init sequence; then it becomes 1 there on. I wasn't seeing success at 1, so I put back to 10, and it blinked. See can.cpp comments.